lib/resources/etc_libuser.rb

```diff
@@ -0,0 +1,108 @@
+require 'utils/simpleconfig'
+require 'utils/object_traversal'
+
+module Inspec::Resources
+  class EtcLibUser < Inspec.resource(1)
+    name 'etc_libuser'
```

Comment:
On what operating systems should this resource run? We should add `supports` line(s) accordingly, such as:

```
supports os_family: 'linux'
```

... that way it won't attempt to run on Windows machines.

Reply:
Natively, it should run on Linux systems and I am happy to make this change. However, I wrote the resource such that it just checked for the config path rather than operating system because it would allow you to evaluate the configuration file on any system (since it does not require any OS-specific commands). This may not be something you want to support, but, I know it was a use case for some vendors when I worked on the OVAL project. Just let me know either way.

Reply:
OK, that sounds reasonable for now, though I'm not sure it makes sense to me. :)  If we don't want to limit it to certain operating system families, I'd definitely like to see the logic to try and pull from the environment variable removed so this focuses on either the sane default, or whatever path the user supplies.